### PR TITLE
Copy over the included vpm bash completions

### PR DESCRIPTION
**Pull request by keshto**

### CODE REVIEW

**Comment by sgn** [comment]
Please bump `revision` and reword the commit msg.

**Comment by Chocimier** [comment]
It's about commit title in git rather than github PR title. Also single commit, please. You may find squash and force push useful.

File: srcpkgs/vpm/template [hunk 1]
```diff
@@ -12,4 +12,5 @@ checksum=56994ba90aaf9055d0c20a7fbbce7faf6b6d245fba551a427312114f7cb78ae4
 
 do_install() {
 	vbin vpm
+	vinstall bash-completion/completions/vpm 644 usr/share/bash-completion/completions vpm
```
Comment (ericonr) [inline srcpkgs/vpm/template:15]: Please use the `vcompletion` helper for this.

**Comment by keshto** [comment]
This PR can be treated as abandoned and thus closed.

**Pull request closed without merging**